recipes/q/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "1.3.1" %}
+
+package:
+  name: samtools
```

Comment:
that looks wrong ... can we name it maybe `q_peakcaller` or something like that?

Reply:
thanks börn. copy and paste error